whipper/common/encode.py

```diff
@@ -89,3 +89,62 @@ def _tag(self):
         w.save()
 
         self.stop()
+
+
+class EmbedPictureTask(task.Task):
+    description = 'Embed picture to FLAC'
+
+    def __init__(self, track_path, coverart_path):
+        self.track_path = track_path
+        self.coverart_path = coverart_path
+
+    def start(self, runner):
+        task.Task.start(self, runner)
+        self.schedule(0.0, self._embed_picture)
+
+    def _make_flac_picture(self, coverart_filename):
+        """Given a path to a jpeg or png file, return a flac.Picture
+
+        Returns: a valid flac.Picture or None
+        """
+        if not coverart_filename:
+            return None
+
+        try:
```

Comment:
This isn't enough as if the module is missing, the exception is caught and logged but the code below (requiring pillow) is still executed.
Please check my previous comment.

Reply:
Ahh yeah you are right.

Reply:
Should i do something like :
```
try:
    from PIL import Image
except ImportError as e:
    logger.warning("the cover art option won't be honored because "
                   "the 'pillow' module isn't available: %r", e)
    return None
```
in `encode.py` so that it will be simpler and also eliminate Flake8's module imported but unused error ?

Reply:
The reason I prefer putting the check in `command.py` is because if we move it in `encode.py` the `getCoverArt` method will be invoked even when the `pillow` dependency isn't available (and the fetched cover will be discarded).